doc/release/upcoming_changes/29537.change.rst

```diff
@@ -0,0 +1,7 @@
+``unique_values`` for float and complex dtypes may return unsorted data
```

Comment:
it's only complex now, right? also other mentions of `float` in the release notes below